test/module/irohad/consensus/yac/yac_gate_test.cpp

```diff
@@ -59,6 +59,8 @@ class YacGateTest : public ::testing::Test {
     auto prev_hash = Hash("prev hash");
     EXPECT_CALL(*block, prevHash())
         .WillRepeatedly(testing::ReturnRefOfCopy(prev_hash));
+    EXPECT_CALL(*block, hash())
+        .WillRepeatedly(testing::ReturnRefOfCopy(prev_hash));
```

Comment:
Such values can be misleading for people not familiar with the project, as we expect prev hash to be different from block hash. Event though it is possible with current types and interfaces, I suggest not to create such cases.